### DISCUSSION TITLE
Ajout de fonctionnalités pour les pets

### DISCUSSION THREAD

**Discussion by voltou91** [Crownicles/Crownicles#1094] [Idées acceptées / déjà implémentées]
On pourrait imaginer une amélioration de pets grâce au nouveau shop de gemmes, voici différentes propositons:
1) On fait évolué les pets en leur rajoutant juste des étoiles (avec un max de 5 ⭐).
1 BIS) Tout les pets démarrent à 1 ⭐ et on doit les faire monter en étoiles.
2) On fait évolué son pet du genre tu as un pet 2 ⭐ végétarien alors quand tu vas le up ce sera un autre pet végétarien mais 3 ⭐ comme si il évoluait tel un pokemon.
Concernant les propositions du dessus le système de loot de tel ou tel animal pourrait rester le même y aurait peu de ce pet là, même si tout le monde fait évolué son pet bah y aurait toujours des pets rares que peut de personne on looter.

3) Où sinon un système de niveau indépendant des étoiles. Les niveaux augmentent avec les mini-event des pets et genre tout les paliers de 5 level on pourrait dire que ça offre un boost de +0.5*le nombre d'étoiles% ( Un pet 1 ⭐= +0.5%, Un pet  5 ⭐ = +2.5%) de récompenses lors d'un mini-event et l'exp des levels seraient exponentielle bien évidemment et pour monter en exp hors mini-event pour les pets on pourrait faire un jus de diamant qui serait dans le shop des gemmes et qui couterait très cher mais qui donne énormément d'exp.

**Comment by Iris-for** [comment]
J ai pas compris perso

**Reply by voltou91** [reply]
Ce sont différentes propositions qui pourraient affecter les pets. Ici je propose soit de faire monter en étoile un pet grâce au shop des gemmes, soit de faire "évolué" un animal en un autre animal avec une étoile en plus mais avec le même type de régime (végétarien devient végétarien), où même de créé un système de level indépendant des étoiles qui peut donner un boost de quelques pourcents sur les loots des mini-event, dans cette dernière proposition on ne pourrait donc pas augmenter les étoiles de son pet.

**Comment by Kamardine1** [comment]
Sinon autre proposition : Pouvoir acheter des pets avec des gemmes.
Bien-sûr c'est pas toi qui choisis, ca sera comme les items ça sera random.

**Comment by Chevalier2** [comment]
La meilleure proposition est selon moi d'ajouter des niveaux aux pets. Pas forcément comme la 3e proposition, mais au moins rajouter ça et qu'on puisse augmenter les niveaux des pets avec les gemmes.

**Reply by voltou91** [reply]
Je pense aussi que rajouter des niveaux aux pets c'est la meilleur solution. Cependant j'opterais pour que les niveaux influence très légèrement les mini-event (ou même les events) sur leurs récompense. Et l'idée que les étoiles influe aussi sur le bonus que le pet peut nous donner est la meilleure je pense. Comme ça niveaux et rareté du pet se complètent pour donner un mini bonus/avantage lors d'un mini-event.
